src/Microsoft.Azure.WebJobs.Host/Indexers/FunctionIndexer.cs

```diff
@@ -376,6 +383,7 @@ internal async Task IndexMethodAsyncCore(MethodInfo method, IFunctionIndexCollec
                 IsDisabled = disabled,
                 HasCancellationToken = hasCancellationToken,
                 TimeoutAttribute = TypeUtility.GetHierarchicalAttributeOrNull<TimeoutAttribute>(method) ?? defaultTimeout,
+                RetryAttribute = TypeUtility.GetHierarchicalAttributeOrNull<RetryAttribute>(method) ?? defaultRetry,
```

Comment:
This means that the default host level retry might be linear, while individual functions can have an exponential. That's good, just calling it out